luarules/gadgets/unit_collision_damage_behavior.lua

```diff
@@ -18,52 +18,67 @@ local fallDamageMagnificationFactor = 14 --the multiplier by which default engin
 local groundCollisionDefID = Game.envDamageTypes.GroundCollision
 local objectCollisionDefID = Game.envDamageTypes.ObjectCollision
 local objectCollisionVelocityThreshold = 3.3 --this defines how fast a unit has to be moving in order to take object collision damage
-local maxImpulseThreshold = 0.08 --incease this to make units move less from impulse. This defines the max impulse damage a unit can take expressed in a percentage multiplier
-local cooldownsFrameThreshold = Game.gameSpeed --this is how long in frames before a unit's maximum impulse limit resets.
+local maxImpulseProportion = 0.04 --incease this to make units move less from impulse. This defines the max impulse damage allowed a unit can take relative to its mass.
```

Comment:
```suggestion
local massToImpulseFactor = 25 -- Decrease this to make units move less from impulse. This defines the max allowed impulse damage a unit can take relative to its mass.
```
This seems to be what this actually represents? Change it from a divisor to a multiplier for the 0.04 -> 25 change.

Reply:
I did this and adjusted the variables for optimal results